src/pages/OnboardingPersonalDetails/BaseOnboardingPersonalDetails.tsx

```diff
@@ -36,6 +37,7 @@ function BaseOnboardingPersonalDetails({currentUserPersonalDetails, shouldUseNat
     const {translate} = useLocalize();
     const {isSmallScreenWidth} = useWindowDimensions();
     const {shouldUseNarrowLayout} = useOnboardingLayout();
+    const {inputCallbackRef} = useAutoFocusInput();
```

Comment:
Basically it focuses `First Name` input here just after transition is ended and everything is set up, doing it by ref instead of `autoFocus={true}` as it was before avoid focus to go to `Composer` in Concierge chat